<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shoppingCartDlg</class>
 <widget class="QDialog" name="shoppingCartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>shoppingCartDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget{
border-image: url(:/masterDesktop/Resources/shopping cart_bg.png);
}
#wgt_top{
border-bottom:1px solid #00947E;
}
#wgt_bottom{
border-top:1px solid #00947E;
}
#listWidget{
border:none;
background:transparent;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_top" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>55</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>55</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>18</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#FFFFFF;</string>
           </property>
           <property name="text">
            <string>已选商品</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>297</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_clearCart">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:12px;
color:#2EB4AF;
border:none;</string>
           </property>
           <property name="text">
            <string>清空购物车</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="minimumSize">
         <size>
          <width>460</width>
          <height>225</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>460</width>
          <height>225</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget{outline:0px;}
QListView::item:hover, QListView::item:selected{
background:transparent;
}
QScrollBar:vertical {
       width: 13px;
    background:rgba(0,0,0,0%);
    padding-top:13px;
    padding-bottom:13px;
   }
   QScrollBar::handle:vertical {
       
	border-image: url(:/masterDesktop/Resources/slider.png);
   }
QScrollBar::add-line:vertical
{
    border:0px;
   height:13px;
}

QScrollBar::sub-line:vertical
{
  border:0px;
height:13px;
}
 QScrollBar::up-arrow:vertical{     
    border:0px;
width:13px;
height:13px;
	border-image: url(:/masterDesktop/Resources/up.png);
   }
QScrollBar::down-arrow:vertical{
    border:0px;
width:13px;
height:13px;
border-image: url(:/masterDesktop/Resources/down.png);
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
  background-color:#08201e;
}
</string>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_bottom" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>85</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>85</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#btn_calculate{
font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#151515;
background:#2EB4AF;
border:none;
border-radius:4px;
}
#btn_calculate:pressed{
margin:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="bottomMargin">
          <number>18</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>138</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lbl_num">
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:16px;
color:#EEEEEE;</string>
           </property>
           <property name="text">
            <string>共计0件商品</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_sum">
           <property name="minimumSize">
            <size>
             <width>128</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>128</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font-family:&quot;微软雅黑&quot;;
font-size:20px;
color:#00D5CF;</string>
           </property>
           <property name="text">
            <string>¥0.00</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_calculate">
           <property name="minimumSize">
            <size>
             <width>78</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>78</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>去结算</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="masterDesktop.qrc"/>
 </resources>
 <connections/>
</ui>
